### PR TITLE
New package: Exiv2.Exiv2 version 0.28.2

### LINKED ISSUE #139958: [Package Request]: exiv2
### How can we help?

I would like someone else to build the manifest.

### Please read and ensure the following

- [X] The installer meets the above requirements

### Please provide the following information

Download Page Url: https://github.com/Exiv2/exiv2/releases
Publisher: Exiv2 Team 
Package Name: Exiv2 
Description: Exiv2 is a command-line utility to read, write, delete and modify Exif, IPTC, XMP and ICC image metadata.
Package Version: v0.28.2
Installer URL:  https://github.com/Exiv2/exiv2/releases

This is a related issue from project github: https://github.com/Exiv2/exiv2/issues/2927

### PR DESCRIPTION
**Pull request by Exorcism0666**
- [x] Resolve: #139958

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/139966)

### CODE REVIEW

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> No errors to post.

(Automated response - build 792.)

**Comment by stephengillie** [comment]
Manual Validation ended with: 
image

File: manifests/e/Exiv2/Exiv2/0.28.2/Exiv2.Exiv2.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: Exiv2.Exiv2
+PackageVersion: 0.28.2
+InstallerType: zip
+NestedInstallerType: portable
+NestedInstallerFiles:
+- RelativeFilePath: exiv2-0.28.2-2019msvc64\bin\exiv2.exe
+Installers:
```
Comment (stephengillie) [inline manifests/e/Exiv2/Exiv2/0.28.2/Exiv2.Exiv2.installer.yaml:10]:
```suggestion
Dependencies:
  PackageDependencies:
   - PackageIdentifier: Microsoft.VCRedist.2015+.x64
Installers:
```

(Automated response - build 792.)

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> No errors to post.

(Automated response - build 796.)

**Comment by stephengillie** [comment]
Manual Validation ended with:
> PS C:\Users\User\Desktop> C:\Users\User\AppData\Local\Microsoft\WinGet\Links\exiv2.exe PS C:\Users\User\Desktop> C:\Users\User\AppData\Local\Microsoft\WinGet\Packages\Exiv2.Exiv2__DefaultSource\exiv2-0.28.2-2019msvc64\bin\exiv2.exe exiv2.exe: An action must be specified exiv2.exe: At least one file is required Usage: exiv2.exe [ option [ arg ] ]+ [ action ] file ...  Image metadata manipulation tool.

(Automated response - build 795.)

**Comment by Exorcism0666** [comment]
## For moderators

This account is automated by Github Actions and the source code was created by CoolPlayLin. If you have any questions about any pull request, don't hesitate to ping Exorcism0666.
>[!warning]
>With the new version of Komac, numerous Pull Requests may contain bugs or unwanted elements. If such issues arise, please tag russellbanks so that these bugs can be addressed in future updates.

>[!important]
>Please carefully review these Pull Request before merging. If it is a Pull Request for removing incorrect content and the URLs are issue, free upon manual checking, please close this Pull Request directly. (It is best to inform Exorcism0666 of the closure.)

良い一日をお過ごしください！

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Comment by Trenly** [comment]
@Exorcism0666 - Support for zipped binaries has now been added and is available in the pipelines. This will require you to update your submission to ManifestVersion: 1.9.0 and set ArchiveBinariesDependOnPath: true in the installer node. To keep things clean, this PR is being closed. Please submit a new PR for the latest version of this package. Thank you for your contributions!

**Pull request closed without merging**